combat-trainer.lic

```diff
@@ -65,6 +70,27 @@ class SetupProcess
 
   private
 
+  def blow_warhorn()
+    return unless @warhorn
+    return if Time.now - @last_warhorn < 600
+
+    case bput("tap my #{@warhorn}", 'inside your', 'that you are wearing', 'I could not find what you were referring to.')
```

Comment:
Would using `DRCI.wearing?` simplify this?

Reply:
Done!